### PR TITLE
Nostr-specific Private Key from Deterministic Wallet Signature (Sign-in-With-X)

### PR DESCRIPTION
**Pull request by 0xc0de4c0ffee**
Draft implementation for `NIP-XX: Nostr-specific Private Key from Deterministic Wallet Signature (Sign-in-With-X)` https://github.com/nostr-protocol/nips/pull/268

### CODE REVIEW

**Comment by fiatjaf** [comment]
I don't understand this and am not convinced it is a good thing.

**Comment by sshmatrix** [comment]
Sorry, I made a mistake while housekeeping and deleted branch under PR. I have restored it. Apologies for the small mess up

File: lib/.DS_Store
Comment (offbyn) [inline lib/.DS_Store:1]: remove .DS_Store

**Comment by BilligsterUser** [comment]
@fiatjaf Can we close this?

**Comment by 0xc0de4c0ffee** [comment]
Thanks for your time & feedback.. 

Closing this for now/ 🙏 🖖

**Pull request closed without merging**